css/css-text/white-space/pre-wrap-010.html

```diff
@@ -1,23 +1,24 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>CSS Text level 3 Test: single preserved white space at the end of with white-space:pre-wrap overflow-wrap:break-spaces break-word</title>
+<title>CSS Text level 3 Test: single preserved white space at the end of the line with white-space:break-spaces overflow-wrap:break-word</title>
 <link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
 <link rel="help" href="https://drafts.csswg.org/css-text-3/#white-space-phase-2">
 <link rel="help" href="https://drafts.csswg.org/css-text-3/#overflow-wrap-property">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#white-space-property">
 <link rel="match" href="reference/pre-wrap-001-ref.html">
-<meta name="assert" content="a single preserved white space at the end of the line is wrapped when the white-space property is set to pre-wrap and overflow-wrap is break-spaces break-word.">
+<meta name="assert" content="a single preserved white space at the end of the line is wrapped when the white-space property is set to break-spaces and overflow-wrap is break-word.">
```

Comment:
What's the difference between this test and the one defined in overflow-wrap-break-word-003.html ?

Reply:
I don't think there's a difference. I don't remember exactly, but I think we ended up with two tests because the first one I submitted took so long to get review that I forgot about it, and wrote a new one, and eventually both were merge.

Given that they test the same thing is slightly different ways, I think it is fine to have a little bit of redundancy, even if it isn't really necessary.